config/t8_fortran.m4

```diff
@@ -0,0 +1,34 @@
+dnl T8_CHECK_FORTRAN
+dnl This functions checks some properties of Frotran modules and
+dnl whether specific directory for the Fortran module files has been specified or not.
+dnl 
+dnl A directory may be specified by the option --with-moddir=<directory_path>
+dnl This option is only of relevance if the option --enable-fortran has been chosen,
+dnl since only in this case Fortran codes will be compiled 
+dnl
+AC_DEFUN([T8_CHECK_FORTRAN], [
+
+dnl Check if a directory has been specified which will hold the module files
+T8_ARG_WITH([moddir],
+  [if Fortran modules will be built, this option specifies an explicit directory which should hold the module files (use --with-moddir=<desired/path/for/modules>)],
+  [MODDIR])
```

Comment:
Brauchen wir das vor der Abfrage, ob fortran enabled ist?

Reply:
The macro `T8_ARG_WITH`  is a wrapper for the corresponding SC-Macro which checks the option and also defines an `AM_CONDITIONAL` (in this case the conditional `T8_WITH_MODDIR`). It is not permitted to call `AM_CONDITIONAL` conditionally within an if-statement. Therefore, we have to call this (unconditionally) before the line `if test "x$T8_ENABLE_FORTRAN" != xno ; then`